operations/kubernetes/Retrieve_Cluster_Health_Information_Using_Kubernetes.md

```diff
@@ -21,134 +22,81 @@ ncn-w002   Ready    <none>                 8d    v1.20.13
-api-gateway-6fffd4d854-tqx8v            1/1       Running   1          11d
-api-gateway-database-8555685975-xw5gm   1/1       Running   1          11d
-cray-ars-6466bcf77d-s99rq               1/1       Running   1          11d
-cray-bss-7589d7459f-ttwf4               1/1       Running   1          11d
-cray-capmc-6bf5855b78-np72d             1/1       Running   1          11d
-cray-datastore-768576677d-gpnl7         1/1       Running   1          11d
-cray-dhcp-5fccf85696-wrqxb              1/1       Running   1          11d
-cray-tftp-885cc65c4-568jz               2/2       Running   2          11d
-kvstore-1-56fdb6574c-ts79v              1/1       Running   1          11d
-kvstore-2-694bc7567b-k99tl              1/1       Running   1          11d
-kvstore-3-5b658b9bd9-n9dgt              1/1       Running   1          11d
+```bash
+ncn# kubectl get pods -A
 ```
 
-### Retrieve Information about Individual Pods
+### Retrieve a list of healthy pods
 
 ```bash
-ncn# kubectl describe pod POD_NAME -n NAMESPACE_NAME
+ncn# kubectl get pods -A | grep -E 'Completed|Running'
 ```
 
-### Retrieve a List of Healthy Pods
+### Retrieve a list of unhealthy pods
 
 ```bash
-ncn# kubectl get pods -A | grep -e 'Completed|Running'
+ncn# kubectl get pods -A | grep -Ev 'Completed|Running'
 ```
 
-Example output:
+or
 
-```
-ceph-cephfs       cephfs-provisioner-74599ccfcd-bzf5b                1/1     Running      3    2d11h
-ceph-rbd          rbd-provisioner-76c464c567-jvvk6                   1/1     Running      3    2d11h
-cert-manager      cray-certmanager-cainjector-8487d996d7-phbfr       1/1     Running      0    2d11h
-cert-manager      cray-certmanager-cert-manager-d5fb67664-v9sgf      1/1     Running      0    2d11h
-cert-manager      cray-certmanager-webhook-6b58c6bb79-dtpl7          1/1     Running      0    2d11h
-default           kube-keepalived-vip-mgmt-plane-nmn-local-pkbxj     1/1     Running      0    2d11h
-default           kube-keepalived-vip-mgmt-plane-nmn-local-v8w6k     1/1     Running      0    2d11h
-default           kube-keepalived-vip-mgmt-plane-nmn-local-zzc4n     1/1     Running      1    2d11h
-istio-system      grafana-ff8b4b964-grg8m                            1/1     Running      0    2d11h
-istio-system      istio-citadel-699fc7bcf-rn99p                      1/1     Running      0    2d11h
-istio-system      istio-ingressgateway-59bf97fbc5-h6x4w              1/1     Running      0    2d11h
-istio-system      istio-ingressgateway-hmn-5d7777c75-2c5nz           1/1     Running      0    2d11h
-istio-system      istio-ingressgateway-hmn-5d7777c75-dh4fl           1/1     Running      0    2d11h
-istio-system      istio-init-crd-10-1.2.10-wgg99                     0/1     Completed    0    2d11h
-istio-system      istio-init-crd-11-1.2.10-dcp4h                     0/1     Completed    0    2d11h
+```bash
+ncn# kubectl get pods -A | grep -E 'Creating|ImagePull|Error|Init|Crash'
```

Comment:
This seems out of place.  Prob goes with the "or" and might be better at line 50?